### ISSUE TITLE
Foundry 0.9: Run Encounter using Compendium actor fails

### ISSUE THREAD

**Issue by spetzel2020** [spetzel2020/quick-encounters#81]
Error:
```
QuickEncounter.js:1050 Uncaught (in promise) TypeError: game.actors.importFromCollection is not a function
    at Function.getActor (QuickEncounter.js:1050)
    at QuickEncounter.generateFullExtractedActorTokenData (QuickEncounter.js:1083)
    at async QuickEncounter.run (QuickEncounter.js:909)
```
Needs to use `importFromCompendium` . See PR #80 for fix.

**Comment by spetzel2020** [comment]
Fixed in 0.9.10a

**Comment by spetzel2020** [comment]
Now getting a new error with my testing:
```
foundry.js:11467 Uncaught (in promise) Error: The undefined Document type provided by Compendium undefined is incorrect for this Collection
    at Map.importFromCompendium (foundry.js:11467)
    at Function.getActor (QuickEncounter.js:1058)
    at QuickEncounter.generateFullExtractedActorTokenData (QuickEncounter.js:1092)
    at async QuickEncounter.run (QuickEncounter.js:913)
```

**Comment by spetzel2020** [comment]
Fixed typo in PR#80: `importFromCollection` first parameter is `pack` object, not packName.
Released in 0.9.10
